src/controller/BUILD.gn

```
@@ -18,6 +18,22 @@ import("${chip_root}/src/lib/lib.gni")
 import("${chip_root}/src/platform/device.gni")
 import("${chip_root}/src/platform/python.gni")
 
+# ICD Client sources and configurations
+source_set("icdclient") {
```

```suggestion
source_set("client") {
```
Just to avoid having `icd:icdclient` when it will be used a dependency

---

rename it to icd instead in controller. thanks